generators/client/templates/angular/src/main/webapp/app/account/activate/_activate.component.ts

```diff
@@ -16,18 +17,27 @@ export class ActivateComponent implements OnInit {
     modalRef: NgbModalRef;
 
     constructor(
+        <%_ if (enableTranslation) { _%>
+        private jhiLanguageService: JhiLanguageService,
```

Comment:
update the regex in https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-base.js#L1094 to strip this line and pass components through `copyJs` method